src/layout/List/ListComponent.tsx

```diff
@@ -53,6 +54,11 @@ export const ListComponent = ({
     }
   };
 
+  const tableHeadersValues = tableHeaders;
```

Comment:
Assigning an object to a variable still just references the original object, and updates to it will update the source/reference as well. So this would work as intended:
```suggestion
  const tableHeadersValues = {...tableHeaders};
```